include/seqan3/std/charconv

```diff
@@ -859,6 +859,23 @@ inline std::to_chars_result to_chars(char * first, char * last, value_type value
     return seqan3::detail::to_chars_itoa(first, last, value, is_signed<value_type>());
 }
 
+//!\brief std::to_chars overload for floating point via a stringstream.
+//!\ingroup charconv
+template <seqan3::FloatingPoint floating_point_type>
+inline std::to_chars_result to_chars(char * first, char * last, floating_point_type value, int = 10) noexcept
```

Comment:
Parameter 4 has no name, but a default value? Why?

Reply:
I thought it has to be consistent with the integral overload, but actually I do not support a diffferent base then 10 so I removed the parameter.